### PR TITLE
SOV-3390: reimplement increasing debt without adding collateral

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-3390

### CODE REVIEW

File: apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/AdjustLoanForm.tsx [hunk 1]
```diff
@@ -674,6 +682,42 @@ export const AdjustLoanForm: FC<AdjustLoanFormProps> = ({ loan }) => {
     [isValidDebtAmount],
   );
 
+  const [drawDown, setDrawDown] = useState(Decimal.ZERO);
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/AdjustLoanForm.tsx]: It looks good but pls extract it to a custom hook that accepts `loan`, `isAddCollateralTab` and returns `drawdown` and `maxBorrow`.

File: apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/utils.ts
Comment (tiltom) [inline apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/utils.ts:1]: The filename should be `AdjustLoanForm.utils.ts`.

**Review by tiltom — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/utils.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  ).contract(getProvider(defaultChainId));
+  const loanAddress = (
+    await getTokenDetails(normalizeTokenWrapped(loanToken), defaultChainId)
+  ).address;
+  const collateralAddress = (
+    await getTokenDetails(
+      normalizeTokenWrapped(collateralToken),
+      defaultChainId,
+    )
+  ).address;
```
Comment (Rickk137) [inline apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/utils.ts]: you can use `Promise.all` for these async tasks

File: apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/utils.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  collateralToken: SupportedTokens,
+  durationInSeconds: number,
+): Promise<Decimal> => {
+  const contract = (
+    await getLoanTokenContract(loanToken, defaultChainId)
+  ).contract(getProvider(defaultChainId));
+
+  const collateralTokenAddress = (
+    await getTokenDetails(collateralToken, defaultChainId)
+  ).address;
```
Comment (Rickk137) [inline apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/utils.ts]: you can use `Promise.all` for these async tasks

**Review by Rickk137 — CHANGES_REQUESTED** [review]

**Review by soulBit — APPROVED** [review]

**Review by Rickk137 — APPROVED** [review]

**Pull request merged**